### DISCUSSION TITLE
BBN working on Orangepi?

### DISCUSSION THREAD

**Discussion by bektorkhan** [bareboat-necessities/lysmarine_gen#408] [Q&A]
I have always used Orangepi for my projects.. and I have a handful..
Waiting for Opi Zero 2W (Alwinner H618) and CM4 (RK3566)both with 4G Ram.. any major issues to expect except WiringPi that is replaced with WiringOpi-
I'll try it and Post the result

**Comment by mgrouch** [comment]
You will have to fork the code and update the build scripts to base it from some Armbian Bookworm image which runs on your hardware. And build your own image on circleci

**Reply by bektorkhan** [reply]
Thanks
In the meantime I'll try it manually on a OPI Zero 2W 2Gb.. see what works.. 
Circleci looks promising will save some time!
Looking for the smartest way to run HAOS, keeping it in a VM.
I've been trying to make ESPHome devices and MQTT in Esphome takes to much Ram. Need to implement signalK in ESPHome!
